### PR TITLE
👷🔧 Add CI workflow for automatic linting and formatting

### PR DESCRIPTION
**Pull request by drillskibo**
This workflow sets an action to be ran automatically on push. It uses the latest version of Ubuntu, setups Deno, then runs the linter and the formatter.

### CODE REVIEW

**Comment by drillskibo** [comment]
Closes #62.

**Comment by drillskibo** [comment]
I'm not sure if `--check` flag should be used or not.
Maybe @omar2205 knows ?

**Comment by omar2205** [comment]
> I'm not sure if `--check` flag should be used or not. Maybe @omar2205 knows ?

running `deno fmt` will try and fix (write), using `--check` will throw an error

File: .github/workflows/deno-ci.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+name: Deno CI
+on: push
```
Comment (omar2205) [inline .github/workflows/deno-ci.yml:2]: Better to listen to PRs too. 
`on: [push, pull_request]`

**Review by omar2205 — CHANGES_REQUESTED** [review]

File: .github/workflows/deno-ci.yml (hunk 1 above)
Comment (omar2205) [reply .github/workflows/deno-ci.yml:2]: This way, we can see a failed check in the PR if `deno fmt --check` fails

**Pull request merged**